package.json

```diff
@@ -71,7 +72,8 @@
   },
   "license": "Apache-2.0",
   "scripts": {
-    "prepublishOnly": "python build/checkversion.py && python build/all.py --force"
+    "prepublishOnly": "python build/checkversion.py && python build/all.py --force",
+    "testTypescript": "tsc -p tsconfig.json"
```

Comment:
Instead of putting this into package.json and calling "npm run ...", you should call "tsc" directly.  We have helpers for this:

```py
  base = shakaBuildHelpers.get_source_base()
  tsc = shakaBuildHelpers.get_node_binary('typescript', 'tsc')
  command_line = tsc + ['-p', 'tsconfig.json']
```

The reason for the way we do things is that we have Python-based internal infrastructure tools that import these python scripts as modules.  Some day, we may migrate to something more familiar to the node.js world, but until we can also replace those internal tools, it helps to be consistent.  Thanks!